.changeset/quick-bottles-march.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Allow content collections to run in browser
```

Comment:
Is the changeset correct? I don't think this PR is around content collections

Reply:
Yeah it is, look at the stackblitz example.

Reply:
But title and description of the PR talk about csssec, I'm confused.

Reply:
View Transitions has both a server and client part to it. The client part has no issues, works fine. The server part depends on `cssesc`. You don't load that code in the client. Unless you import `astro:content`, in which that comes along for the ride.